frontend/src/crypto.test.js

```diff
@@ -8,9 +8,19 @@ test('subtracts 3 and 2 to equal 1', () => {
   expect(subtract(3, 2)).toBe(1);
 });
 
-test('generate key', async () => {
-  const key = await generateAesKey();
+test('generate key, encrypt and decrypt', async () => {
+  plaintext = "blabla42";
+
+  const key = await generateKey();
+  const {
+    privateKey,
+    publicKey
+  } = await key;
```

Comment:
nit: awaiting here isn't needed since `key` is available synchronously.

```suggestion
  } = key;
```

Reply:
thanks!